src/app/components/history/history.component.html

```diff
@@ -1,19 +1,36 @@
 <div class="container-fluid">
+  <div class="row mb-2">
+    <div class="btn-group btn-group-toggle">
+      <label class="btn btn-secondary" for="filter_all" [class.active]="filter === 'all'">
+        <input type="radio" [ngModel]="filter" value="all" name="options" id="filter_all" (ngModelChange)="filterHistory($event)" autocomplete="off"> {{ 'All' | translate }}
```

Comment:
Can you make the lines short enough to fit into the normal Github review window, or maybe limit to 100 characters?